oauth2l-go/src/oauth2l/oauth2l.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"oauth2client"
+	"strings"
+)
+
+const (
+	// Common prefix for google oauth scope
+	scopePrefix = "https://www.googleapis.com/auth/"
+)
+
+func help() {
+	fmt.Println("Usage: oauth2l --json <secret.json> " +
+		"{fetch|header|token|reset} scope1 scope2 ...")
+}
+
+func fetch(token *oauth2client.Token) {
+	fmt.Println(token.AccessToken)
+}
+
+func header(token *oauth2client.Token) {
+	fmt.Printf("Authorization: %s %s\n", token.TokenType, token.AccessToken)
+}
+
+func token(token *oauth2client.Token) {
+	jsonStr, err := json.MarshalIndent(token, "", "  ")
+	if err != nil {
+		panic("Failed to covert token to json.")
+	}
+	fmt.Println(string(jsonStr))
+}
+
+// Join scopes with a space
+func scopesToString(scopes []string) string {
+	return strings.Join(scopes, " ")
+}
+
+func main() {
+	jsonFile := flag.String("json", "", "Path to secret json file.")
+	helpFlag := flag.Bool("help", false, "Print help message.")
+	flag.BoolVar(helpFlag, "h", false, "")
+
+	flag.Parse()
+
+	if *helpFlag || len(flag.Args()) < 2 {
+		help()
+		return
+	}
+
+	commands := map[string]func(*oauth2client.Token){
+		"fetch":  fetch,
+		"header": header,
+		"token":  token,
+	}
+	secretBytes, err := ioutil.ReadFile(*jsonFile)
+	if err != nil {
+		fmt.Printf("Failed to read file %s.\n", *jsonFile)
+		return
+	}
+
+	if cmdFunc, ok := commands[flag.Args()[0]]; ok {
```

Comment:
Change this one to 

  if ; !ok {
    help()
    return
  }

  ....

Otherwise the ident is too deep to read.